README.md

```diff
@@ -1,555 +1,329 @@
-    questions and [GitHub issues](https://github.com/ReactiveCocoa/ReactiveCocoa/issues?labels=question&state=closed)
- 1. The rest of the [Documentation][] folder
- 1. [Functional Reactive Programming on iOS](https://leanpub.com/iosfrp/) 
-    (eBook)
- 
-If you have any further questions, please feel free to [file an issue](https://github.com/ReactiveCocoa/ReactiveCocoa/issues/new). 
-
 ## Introduction
 
 ReactiveCocoa is inspired by [functional reactive
 programming](http://blog.maybeapps.com/post/42894317939/input-and-output).
 Rather than using mutable variables which are replaced and modified in-place,
-RAC provides signals (represented by `RACSignal`) that capture present and
-future values.
-
-By chaining, combining, and reacting to signals, software can be written
-declaratively, without the need for code that continually observes and updates
-values.
-
-For example, a text field can be bound to the latest time, even as it changes,
-instead of using additional code that watches the clock and updates the
-text field every second.  It works much like KVO, but with blocks instead of
-overriding `-observeValueForKeyPath:ofObject:change:context:`.
-
-Signals can also represent asynchronous operations, much like [futures and
-promises][]. This greatly simplifies asynchronous software, including networking
-code.
-
-One of the major advantages of RAC is that it provides a single, unified
-approach to dealing with asynchronous behaviors, including delegate methods,
-callback blocks, target-action mechanisms, notifications, and KVO.
-
-Here's a simple example:
-
-```objc
-// When self.username changes, logs the new name to the console.
-//
-// RACObserve(self, username) creates a new RACSignal that sends the current
-// value of self.username, then the new value whenever it changes.
-// -subscribeNext: will execute the block whenever the signal sends a value.
-[RACObserve(self, username) subscribeNext:^(NSString *newName) {
-	NSLog(@"%@", newName);
-}];
-```
+RAC offers “event streams,” represented by the [`Signal`][Signals] and
+[`SignalProducer`][Signal producers] types, that send values over time.
 
-But unlike KVO notifications, signals can be chained together and operated on:
-
-```objc
-// Only logs names that starts with "j".
-//
-// -filter returns a new RACSignal that only sends a new value when its block
-// returns YES.
-[[RACObserve(self, username)
-	filter:^(NSString *newName) {
-		return [newName hasPrefix:@"j"];
-	}]
-	subscribeNext:^(NSString *newName) {
-		NSLog(@"%@", newName);
-	}];
-```
+Event streams unify all of Cocoa’s common patterns for asynchrony and event
+handling, including:
+
+ * Delegate methods
+ * Callback blocks
+ * `NSNotification`s
+ * Control actions and responder chain events
+ * [Futures and promises](https://en.wikipedia.org/wiki/Futures_and_promises)
+ * [Key-value observing](https://developer.apple.com/library/mac/documentation/Cocoa/Conceptual/KeyValueObserving/KeyValueObserving.html) (KVO)
```

Comment:
That's a good point, but might be a little confusing. Hopefully “responder chain events” covers it sufficiently.